test/dummy/.byebug_history

```diff
@@ -0,0 +1,5 @@
+exit
+finish
+puts @sections_content
+next
+puts @sections
```

Comment:
Please remove this file as we never commit `debugger/byebug` generated files into code.